server/plugin/command.go

```diff
@@ -85,6 +88,99 @@ func (p *Plugin) getGithubClient(userInfo *GitHubUserInfo) *github.Client {
 	return p.githubConnect(*userInfo.Token)
 }
 
+func (p *Plugin) getMutedUsernames(userInfo *GitHubUserInfo) []string {
+	mutedUsernameBytes, err := p.API.KVGet(userInfo.UserID + "-muted-users")
+	if err != nil {
+		return nil
+	}
+	mutedUsernames := string(mutedUsernameBytes)
+	var mutedUsers []string
+	if len(mutedUsernames) == 0 {
+		return mutedUsers
+	}
+	mutedUsers = strings.Split(mutedUsernames, ",")
+	return mutedUsers
+}
+
+func (p *Plugin) handleMuteList(args *model.CommandArgs, userInfo *GitHubUserInfo) string {
+	mutedUsernames := p.getMutedUsernames(userInfo)
+	return "Your muted usernames: " + strings.Join(mutedUsernames, ", ")
+}
+
+func (p *Plugin) handleMuteAdd(args *model.CommandArgs, username string, userInfo *GitHubUserInfo) string {
+	mutedUsernames := p.getMutedUsernames(userInfo)
+	var mutedUsers string
+	if len(mutedUsernames) > 0 {
+		// , is a character not allowed in github usernames so we can split on them
+		mutedUsers = strings.Join(mutedUsernames, ",") + "," + username
```

Comment:
![Screenshot from 2020-09-15 10-08-11](https://user-images.githubusercontent.com/16541325/93183531-60f4ab80-f73b-11ea-9b07-e6546dc51b17.png)
Users are able to add the same username multiple times. 2/5 that we should output a message stating that that fact.